### ISSUE TITLE
Figuring out video playback

### ISSUE THREAD

**Issue by TheMrIron2** [TheMrIron2/steamed-hams-channel#4]
Hey guys :wave:

This is a straightforward project on the face of it, but I'm having real difficulties figuring out video playback. I'm not sure how to load and play `hams.mp4` - MPlayer manuals give the frontend documentation but nothing about how the program loads videos. Any help would be appreciated!

**Comment by guest123guest** [comment]
CLOSE IT PLAYBACK IS NOT ALLOWED ON WII

**Comment by TheMrIron2** [comment]
Did you get dropped when you were a kid or something?

**Comment by guest123guest** [comment]
um playback of video can only be done with homebrew

**Comment by TheMrIron2** [comment]
Incorrect. The Photo Channel accepts AVI videos which can be loaded from an SD card.
